### ISSUE TITLE
TypeError: Cannot read property 'source' of null

### ISSUE THREAD

**Issue by langri-sha** [theKashey/rewiremock#102]
I'd like to leave this here to save someone time in case they run into the following.

```
Internal error
TypeError: Cannot read property 'source' of null  
rewiremock/lib/babel/index.js:60:27
```

See [Dynamic import](https://babeljs.io/blog/2019/07/03/7.5.0#dynamic-import-9552-https-githubcom-babel-babel-pull-9552-and-10109-https-githubcom-babel-babel-pull-10109) from Babel 7.5.0 release. Need to ensure `rewiremock/babel` comes before the module transformer like so:

```
module.exports = {
  plugins: [
    '@babel/plugin-proposal-dynamic-import',
    'rewiremock/babel',
    '@babel/plugin-transform-modules-commonjs
  ]
}
```

**Comment by theKashey** [comment]
Anyway - it shall not lead to the error.

**Comment by theKashey** [comment]
Ok, so that's the common issue - I am trying to read data from imports after they were transformed. However, this check could be done during traverse...

**Comment by theKashey** [comment]
Should be fixed in `3.13.8`

**Comment by theKashey** [comment]
Rolling back. There is no way to play nicely together with `@babel/plugin-transform-modules-commonjs` __except__ the right plugin order.

I am adding autodetection for this problem, but rolling the fix back.
